store/tikv/scan_test.go

```diff
@@ -40,6 +42,10 @@ func (s *testScanSuite) SetUpSuite(c *C) {
 	s.store = NewTestStore(c).(*tikvStore)
 	s.recordPrefix = tablecodec.GenTableRecordPrefix(1)
 	s.rowNums = append(s.rowNums, 1, scanBatchSize, scanBatchSize+1, scanBatchSize*3)
+	ctx := context.Background()
+	// Avoid using async commit logic.
+	ctx = context.WithValue(ctx, sessionctx.ConnID, uint64(0))
```

Comment:
maybe old `context.Backgroud` also cannot step into async commit logic too - -?

Reply:
eh...  Maybe I need another way to skip `async` logic.